src/compass/defaults/s1_cslc_geo.yaml

```diff
@@ -44,7 +44,8 @@ runconfig:
           polarization: co-pol
           geocoding:
               flatten: True
-              lines_per_block: 1000
+              lines_per_block: 200
+              columns_per_block: 200
```

Comment:
bursts are pretty rectangular in radar coordinates, would it make more sense to choose rectangular block sizes? I am not sure whether you have done some analysis to understand what is the impact of block size selection on runtime performances and computational memory usage

Reply:
These 2  parameters are no longer used to geocode the SLC. The S1 bursts are small enough to [read in their entirety into memory](https://github.com/opera-adt/COMPASS/blob/b15bffd6adbb9bb0bf20c3185db5a2c10c9a7c7a/src/compass/s1_geocode_slc.py#L220).

Reply:
> These 2 parameters are no longer used to geocode the SLC. The S1 bursts are small enough to [read in their entirety into memory](https://github.com/opera-adt/COMPASS/blob/b15bffd6adbb9bb0bf20c3185db5a2c10c9a7c7a/src/compass/s1_geocode_slc.py#L220).

These 2 parameters are only used for geocoding of metadata.